sematic/resolvers/log_streamer.py

```diff
@@ -319,25 +331,19 @@ def clean_up_streamer(signal_num, frame=None):
             os.close(write_file_descriptor)
 
 
-def _send_signal_or_kill(pid: int, signal_num: int, timeout_seconds: int):
-    """Send the signal to the given pid. If not exited by timeout, send SIGKILL
+def _wait_or_kill(pid: int, timeout_seconds: int):
```

Comment:
The streamer now stops when it sees the "end of transmission" char, rather than when it gets SIGTERM